src/neptune/handler.py

```diff
@@ -538,32 +532,34 @@ def fetch(self):
             * `Namespace handler`
 
         Returns:
-            Value stored in the field or in case of a namespace a dictionary containing all non-Atom fields values.
+            Value stored in the field or, in case of a namespace, a dictionary containing all non-Atom fields values.
```

Comment:
This could be clarified further.

```suggestion
            If called on a field, returns the stored value.
            If called on a namespace, returns a dictionary containing the values of all non-Atom fields.
```